doc/todo/RELEASE

```diff
@@ -9,6 +9,59 @@ Following shell expansions are used in this document:
  ~elektra-ftp .. where ftp files go to
  ~elektra-doc .. where doc flies go to
 
+----------------------------------------------------------------------
+
+This describes what to do for the automated release:
+
+= Tasks before Release =
+
+== Updates ==
+
+- Update doc/COMPILE.md to reflect actually tested setups
+- Write release notes
+
+== Increment Version Number ==
+
+- CMakeLists.txt
+- ~e/scripts/dev/build-debian-package
+- Change VERSION variable in build-server
+- Commit and push the changes
+
+== Check ==
+
+- Everything blue on https://build.libelektra.org/
```

Comment:
This statement was from when we had many build jobs, now actually only https://build.libelektra.org/job/libelektra/job/master/ matters (unless you split it up).